chain/network-primitives/src/types.rs

```diff
@@ -29,6 +29,9 @@ use std::time::Duration;
 use strum::AsStaticStr;
 use tokio::net::TcpStream;
 
+#[cfg(feature = "sandbox")]
+use near_primitives::types::BlockHeightDelta;
+
```

Comment:
This import is only used once. Let's inline this import, and use the full import path inside `NetworkSandboxMessage`.